docs/rst/installation/linux_source_installation.rst

```diff
@@ -0,0 +1,118 @@
+.. _linux_source_installation:
+
+Linux installation from sources
+===============================
+
+This section explains how to build Vulcanexus in Ubuntu Focal.
+Please, start following the `building instructions for ROS 2 Galactic Geochelone <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html>`_ following the steps in the `System setup <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#system-setup>`_ and the `Get ROS 2 code <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#get-ros-2-code>`_ sections.
+
+Get Vulcanexus code
+-------------------
+
+Following ROS 2 building instructions, the ROS 2 code has been already downloaded into ``ros2_galactic/src`` folder.
+Vulcanexus provides several tools and packages to the ROS 2 distribution that have to be included in the workspace:
+
+.. code-block::
+
+    cd ~
+    mv ros2_galactic vulcanexus_galactic
+    cd vulcanexus_galactic
+    wget https://raw.githubusercontent.com/eProsima/vulcanexus/galactic/vulcanexus.repos
+    wget https://raw.githubusercontent.com/eProsima/vulcanexus/galactic/colcon.meta
+    vcs import --force src < vulcanexus.repos
+
+Install Vulcanexus dependencies
+-------------------------------
+
+First, follow the `Install dependencies using rosdep <https://docs.ros.org/en/galactic/Installation/Ubuntu-Development-Setup.html#install-dependencies-using-rosdep>`_ section of ROS 2 installation guide.
+Then, some additional dependencies which are required for the Vulcanexus distribution must be installed.
+Start by adding the Qt 5.15 repository:
+
+.. code-block:: bash
+
+    sudo apt install -y software-properties-common
+    sudo add-apt-repository -y ppa:beineri/opt-qt-5.15.2-focal
+
+Next, install the Vulcanexus required development tools:
+
+.. code-block:: bash
+
+    sudo apt update && sudo apt install -y \
+      libp11-dev \
+      libengine-pkcs11-openssl \
+      libyaml-cpp-dev \
+      openjdk-8-jdk \
+      qt5-default \
+      qt5153d \
+      qt515charts-no-lgpl \
+      qt515graphicaleffects \
+      qt515quickcontrols \
+      qt515quickcontrols2 \
+      qt515quicktimeline-no-lgpl \
+      qt515svg \
+      qt515tools \
+      qt515translations \
+      swig
+
+Build the code in the workspace
+-------------------------------
+
+If any other Vulcanexus or ROS 2 distribution have been installed from binaries, please ensure that the build is done in a fresh environment (previous installation is not sourced).
+This can be checked running the following command:
+
+.. code-block:: bash
+
+    printenv | grep 'VULCANEXUS\|ROS'
+
+The output should be empty.
+Please, be aware that in case you have added the environment sourcing to your ``.bashrc``, you must remove it in order to get a fresh environment.
```

Comment:
```suggestion
Please, be aware that in case the environment sourcing has been added to ``.bashrc``, it must removed it in order to get a fresh environment.
```